mmrazor/models/connectors/base_connector.py

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from abc import ABCMeta, abstractmethod
+
+import torch
+import torch.nn as nn
+
+
+class BaseConnector(nn.Module, metaclass=ABCMeta):
```

Comment:
It's better to inherit from BaseModule?
Then, `self.init_parameters()` could be rewritten by `init_weights()` and `init_cfg`